src/core/Form/SearchInput/SearchInput.tsx

```diff
@@ -18,10 +18,11 @@ import { StatusText } from '../StatusText/StatusText';
 import { LabelText, LabelMode } from '../LabelText/LabelText';
 import { Icon } from '../../Icon/Icon';
 import { baseStyles } from './SearchInput.baseStyles';
+import { InputStatus } from '../types';
 
 type SearchInputValue = string | number | undefined;
 
-type SearchInputStatus = 'default' | 'error';
+type SearchInputStatus = Exclude<InputStatus, 'success'>;
```

Comment:
Interesting, didn't know there was such a thing as Exclude. Seems to have similar behavior with Omit for interfaces.